RecoLocalCalo/HcalRecProducers/src/HcalHitReconstructor.cc

```diff
@@ -554,7 +554,8 @@ void HcalHitReconstructor::produce(edm::Event& e, const edm::EventSetup& eventSe
 	auxflag+=((i->sample(fTS2).capid())<<28);
 	(rec->back()).setAuxHBHE(auxflag);
 
-	(rec->back()).setFlags(0);  // this sets all flag bits to 0
+	// (rec->back()).setFlags(0);  Don't want to do this because the algorithm
+        //                             can already set some flags
```

Comment:
instead of keeping the old revision (code that was there since rev 1)
it would be more appropriate to just say something in line:
```
// set flags in addition to those (possibly) already set by the reco_ algorithm
```

Also, the original design was apparently more factorized: all flags are set in one place.
Can this be preserved in case some rework of hit flagging needs to be done?
(This is not a particularly strong requirement here though.)
Some rework of flagging has been among lingering topics to review/revisit for calo rechit.